SpatialDilatedMaxPooling.lua

```diff
@@ -0,0 +1,72 @@
+local THNN = require 'nn.THNN'
+local SpatialDilatedMaxPooling, parent = torch.class('nn.SpatialDilatedMaxPooling', 'nn.SpatialMaxPooling')
+
+function SpatialDilatedMaxPooling:__init(kW, kH, dW, dH, padW, padH, dilationW, dilationH)
+   parent.__init(self, kW, kH, dW, dH, padW, padH)
+
+   self.dilationW = dilationW or 1
+   self.dilationH = dilationH or 1
+end
+
+function SpatialDilatedMaxPooling:ceil()
```

Comment:
I am using the initialization of MaxPooling. By default it sets to floor mode. From what I understood, you need to explicitly call :floor() or :ceil(). 

I thought similar needs to be done for DilatedPooling as well. Since those were separate function calls.